packages/ui/src/composables/useBem.ts

```diff
@@ -2,7 +2,7 @@ import { computed, Ref, unref, ComputedRef } from 'vue'
 import { __DEV__ } from '../utils/global-utils'
 import isFunction from 'lodash/isFunction.js'
 
-type Key<Prefix extends string, ModifierKey extends string> = `${Prefix}--${ModifierKey}`
+type Key<Prefix extends string, ModifierKey extends string> = `${Prefix}--${ModifierKey | string}`
```

Comment:
```suggestion
type Key<Prefix extends string, ModifierKey extends string> = `${Prefix}--${ModifierKey}`
```

Reply:
In case `ModifierKey` is `propName` (`prop-name` after kebab-cased) this will be incorrect, doesn't it?